### DISCUSSION TITLE
Console のハイライトが機能していない

### DISCUSSION THREAD

**Discussion by yarakos95** [increments/qiita-discussions#282] [要望・アイデア・不具合]
コードブロックで `console` や `shell-session` などの *Console* を利用するとき、うまくハイライトされないようです。

![console in qiita](https://user-images.githubusercontent.com/75878492/186436487-36d1c3e4-09ff-409b-9b7c-94e07f1fefec.png)

Qiita では Rouge を使っているようなので[公式ページ][rouge-demo]を参照しました。公式ページではハイライトが確認できるため、Qiita 側の問題のように思います。

[rouge-demo]: http://rouge.jneen.net/v3.30.0/console/

![rouge demo](https://user-images.githubusercontent.com/75878492/186436878-8835e564-0643-4041-a07e-ebd3683f33f0.png)

**Comment by tanaka-asami415** [comment]
@Yarakashi-Kikohshi 
ご報告ありがとうございます🙏
こちら調査して、順次対応いたします🙇

**Comment by kskwtnk** [comment]
@Yarakashi-Kikohshi 

お待たせしました、こちら対応完了しました！

| 変更前 | 変更後 |
| --- | --- |
| ![](https://qiita-image-store.s3.ap-northeast-1.amazonaws.com/0/88/4a5e75f1-bb6e-d78e-d15c-0b85e235f8f6.png) | ![](https://qiita-image-store.s3.ap-northeast-1.amazonaws.com/0/88/31f4c385-ae0b-41af-5e53-0c699b35dd9c.png) |

**Reply by yarakos95** [reply]
対応ありがとうございます！
ハイライトが機能していることを確認しました。
